scripts/data_files/driver_templates/psa_crypto_driver_wrappers.h.jinja

```diff
@@ -525,7 +525,13 @@ static inline psa_status_t psa_driver_wrapper_sign_hash_start(
     size_t key_buffer_size, psa_algorithm_t alg,
     const uint8_t *hash, size_t hash_length )
 {
-    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+    ( void ) operation;
+    ( void ) key_buffer;
+    ( void ) key_buffer_size;
+    ( void ) alg;
+    ( void ) hash;
+    ( void ) hash_length;
+
```

Comment:
Do we really needs those, though? Looks like the variables are used when calling `mbedtls_psa_sign_hash_start()` below, which looks like it's present in all builds unless I missed something.

Reply:
That's true, I don't think they are needed.